Automattic/vip-go-mu-plugins
# Add custom headers to all ElasticSearch requests

## Description

Fixes [PLAT-932]

Adds following custom header to all requests sent to ElasticSearch

```
X-Client-Site-ID
X-Client-Env
```

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
1. Sandbox a site with VIP Search enabled (e.g. 1120)
1. How can we see whether those headers are really sent out or not? `var_dump()`ing shows they are actually used, but I don't know how we can be 100% sure they are making all their way to ES. 🤔

[PLAT-932]: https://vipjira.atlassian.net/browse/PLAT-932
